geotrek/feedback/forms.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, *args, **kwargs):
                 self.fields["category"].widget = HiddenInput()
                 self.fields["problem_magnitude"].widget = HiddenInput()
                 if settings.SURICATE_WORKFLOW_ENABLED:  # On Workflow
```

Comment:
You can remove this redundant if, and remove "Management" from the comment above on line 83 (legacy from me)